src/vs/workbench/contrib/chat/browser/contrib/chatImplicitContext.ts

```diff
@@ -24,7 +24,7 @@ import { IChatRequestFileEntry, IChatRequestImplicitVariableEntry } from '../../
 import { IChatService } from '../../common/chatService.js';
 import { ChatAgentLocation } from '../../common/constants.js';
 import { ILanguageModelIgnoredFilesService } from '../../common/ignoredFiles.js';
-import { PROMPT_LANGUAGE_ID } from '../../common/promptSyntax/constants.js';
+import { INSTRUCTIONS_LANGUAGE_ID } from '../../common/promptSyntax/constants.js';
```

Comment:
We should keep everything "prompt" here, maybe updating only the `id` getter so the chat variable gets a correct id. These changes were made to make it possible to run a `prompt` file in the chat without prompt input, but it won't make sense to do that for "instructions".